proposals/simd/SIMD.md

```diff
@@ -900,6 +900,30 @@ Lane-wise IEEE `multiplication`.
 
 Lane-wise IEEE `squareRoot`.
 
+### Round to integer above (ceiling)
```

Comment:
Nit: can you reorder this? We have ceil, floor, trunc, nearest in the instruction, this section is ceil, trunc, floor, nearest.

Reply:
Done